src/components/shared/StackSearchField.js

```diff
@@ -53,6 +53,7 @@ class StackSearchField extends React.PureComponent<Props, State> {
     const { searchFieldFocused } = this.state;
     const showIntroduction =
       searchFieldFocused &&
+      searchStrings &&
```

Comment:
I changed the fact that it can be null quite late :( I'm not surprised this slipped out of my attention.